aws-xray/src/test/java/io/opentelemetry/contrib/awsxray/SamplingRuleApplierTest.java

```diff
@@ -120,6 +131,21 @@ void matches() {
           .isTrue();
     }
 
+    @Test
+    void matchesURLFullNewSemCov() {
+      assertThat(applier.matches(newSemCovAttributes, resource)).isTrue();
+
+      // http.url works too
```

Comment:
looks like this comment was copy-pasted from another test, it isn't relevant here

Reply:
updated